### ISSUE TITLE
Add documentation about the new JavaScriptOutputType bridge.json option in the KB article

### ISSUE THREAD

**Issue by fabriciomurta** [bridgedotnet/Bridge#256]
The [Bridge.NET's knowledge base article about config options](http://bridge.net/kb/global-configuration/) should be updated to reflect the new option `JavaScriptOutputType` to `bridge.json`.

The following code should be added to the bottom of the list:
```html
<tr>
<td>outputFormatting</td>
<td>string</td>
<td>Determines whether <b>only formatted</b>, <b>only minified</b> or <b>both</b> versions of the resulting <code>.js</code> files will be output. Default is <code>Both</code>.
<h4>Example</h4>

<pre>outputFormatting: "Formatted"</pre>

<h4>Options</h4>

<dl class="dl-horizontal">

<dt>Both</dt>
<dd>Generates/updates both <b>formatted</b> and <b>minified</b> versions of the <code>.js</code> files when building the project.
<h4>Example</h4>
For default output file naming system, for a class inside the ClassLibrary1 namespace, it will be generated
<code>Class1.js</code>, <code>Bridge.js</code>, <code>Class1.min.js</code> and <code>Bridge.min.js</code> files.
<p>This is the default functionality of Bridge.</p>
</dd>

<dt>Formatted</dt>
<dd>Generates/updates only <b>formatted</b> versions of the <code>.cs</code> files when building the project.
<h4>Example</h4>
For default output file naming system, for a class inside the ClassLibrary1 namespace, it will be generated
<code>Class1.js</code> and <code>Bridge.js</code> files.
</dd>

<dt>Minified</dt>
<dd>Generates/updates only <b>minified</b> versions of the <code>.js</code> files when building the project.
<h4>Example</h4>
For default output file naming system, for a class inside the ClassLibrary1 namespace, it will be generated
<code>Class1.min.js</code> and <code>Bridge.min.js</code> files.
</dd>

</dl>
</td>
</tr>
```

**Comment by fabriciomurta** [comment]
Please, whoever closes the 1.5 milestone ensure this is applied to the KB article. As I couldn't preview the KB article, there might be necessary a little formatting in the code above. It follows the current KB's source format.

**Comment by fabriciomurta** [comment]
Updated and checked on website's KB article.
